server/data/nl/danielsjf.txt

```diff
@@ -4029,3 +4029,190 @@ Ik heb net alle seizoenen van “Friends” in één ruk uitgekeken.
+Er is vooral nood aan extra drinkwater.
+Ik luister graag naar jazz.
+De vooruitgang is stilgevallen.
+We zijn vandaag naar een waterpretpark geweest. 
+Mijn neefje kan al in vier talen tot tien tellen.
+Ken jij alle landen van de wereld?
+Ik kan alle landen van de wereld aanduiden op een blinde kaart.
+De lamp flikkert maar helaas valt er niets aan te doen.
+De flikkerende lamp irriteert mij.
+Ben jij gedoopt?
+De muggenbeet onder mijn oksel jeukt al de hele dag.
+We hebben eten besteld bij de Mongool.
+Hij verkoopt altijd veel flauwekul.
+Met haar valt niet te discussiëren.
+De spion is erin geslaagd om jarenlang onder de radar te blijven.
+Haar ouders hebben een cruise langs de Bahama's gemaakt.
+Ik teken een driehoek op het bord.
+Een vierkant heeft vier gelijke zijden en vier rechte hoeken.
+Zet je de jacuzzi op zevenendertig graden?
+Ik twijfel nog of ik een bubbelbad ga installeren.
+De dynamo van mijn fiets was stuk.
+Gebruik jij je elektrische fiets vaak?
+De ritssluiting zit vast.
+Vergeet je duikbril en snorkel niet in te pakken.
+Zij heeft net haar diploma diepzeeduiken gehaald. 
+Oh lekker, garnalen zijn heerlijk.
+Ik heb haar een cadeau van Rituals gekocht.
+Dove is een bekend merk van Unilever.
+De dief had de kroonjuwelen gestolen.
+Voor het raam bevinden er zich tralies.
+Zijn poging om te ontsnappen was tevergeefs.
+Op welke datum vertrekken jullie?
+Op de betogers waren hun aanhangers talrijk aanwezig.
+Het is niet zo simpel om een revolutie te starten.
+Om lauw water te krijgen moet je zowel de warme als de koude kraan open draaien.
+Toen hij over de ribbels reed maakten zijn banden een vreemd geluid.
+Lawaai heeft een negatieve connotatie.
+Kan je het Kremlin bezoeken?
+De twee bekendste steden van Rusland zijn Moskou en Sint Petersburg.
+Die massage heeft mijn rugklachten toch serieus verminderd.
+Ze durfde jaar benen nooit te tonen vanwege de cellulitis.
+Het is jammer dat meisjes meer last hebben van die hoge schoonheidsidealen.
+Vroeger moesten de monniken boeken handmatig kopiëren.
+Een kopij is het origineel van een kopie.
+Ik ken de auteur van dat boek persoonlijk.
+Zij is een oude jeugdvriendin van mij.
+De jonge advocate zat boordevol ambitie.
+Je kan een woordenboek gebruiken om de betekenis van een woord op te zoeken.
+Je hebt grammatica nodig om woorden te vervoegen.
+Let op met de spelling van dat voltooid deelwoord.
+Sommige werkwoorden hebben een onregelmatige vervoeging.
+Het Latijn is een dode taal wat wilt zeggen dat ze niet meer actief gesproken wordt.
+Zal er een moment komen dat een algoritme mensen beter kan verstaan dan de mensen zelf?
+Mensen blijven continu evolueren.
+Kokosnoten vind je typisch op zonnige eilanden met palmbomen.
+Je kan de knoeier niet verhinderen om te knoeien, maar je kan de knoeiboel wel weghalen bij de knoeier.
+Ga je eindelijk je kamer opruimen?
+Kunnen we hieruit iets leren voor de toekomst?
+OK, mijn idee is niet perfect, maar hoe kunnen we het verbeteren?
```

Comment:
"OK" is een afkorting/ 2 uitspraken ("ok"/"o-kee"), en heeft maar 1 hoofdletter.